### ISSUE TITLE
Build takes a very long time on Arm 32 bit

### ISSUE THREAD

**Issue by coderamen666** [nillerusr/source-engine#324]
The compile requires multiple days worth of time. It prints long warning messages pertaining to sse2neon and is generally inefficient. In addition, when the compile is terminated and restarted, it will not properly pick up where the previous compile left off.

**Comment by nillerusr** [comment]
Waf is properly pick up where the previous compile left off( if terminated with sigint/sighup signals ).
The only thing I can say is that it incorrectly displays the number of remaining files (displaying file index in its internal array instead number of already compiled files).

I think i can remove sse2neon and rewrite sse functions manually. It is very naughty, it can explode even from a compiler update. 
You can pass `--disable-warns` to `./waf configure` as a temporary solution.

**Comment by er2off** [comment]
Agree, dedicated server compilation on 1-core arm one time took 3 days.
